package.json

```diff
@@ -4,30 +4,51 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "dev": "docz dev",
-    "clean": "rm -rf .docz/.cache",
-    "build": "docz build",
-    "lint": "eslint src/**/*.js",
-    "serve": "docz serve",
+    "develop": "yarn run test:generate-output && start-storybook -p 6006",
+    "clean": "rm -rf storybook-static",
+    "build": "yarn run test:generate-output && build-storybook --quiet -c .storybook",
+    "lint": "eslint -c .eslintrc.js . && stylelint --config stylelint.config.js 'src/**/*.scss'",
     "test": "jest src",
+    "test:generate-output": "jest --json --outputFile=.jest-test-results.json",
```

Comment:
What purpose is this serving?

Reply:
Nvm. I see it's used in the develop command. Do you ever run it individually?

Reply:
I'm adding it to the `build` and `develop` scripts and removing it because it's not run individually.